csharp/ql/lib/semmle/code/csharp/PrintAst.qll

```diff
@@ -291,18 +294,6 @@ class ControlFlowElementNode extends ElementNode {
     controlFlowElement = element and
     // Removing implicit expressions
     not isImplicitExpression(element) and
-    // Removing extra nodes that are generated for an `AssignOperation`
```

Comment:
Is the motivation for this PR to get rid of some of the convoluted logic for AST printing (It looks like the deleted code was basically identifying when a synthetic assign operation was made)?